lib/matplotlib/tests/test_axes.py

```diff
@@ -4871,3 +4871,11 @@ def test_scatter_color_masking():
     assert_array_equal(facecolors[1], np.array([0, 0, 0, 1]))
     assert_array_equal(linecolors[1], np.array([0, 0, 0, 1]))
     assert linewidths[1] == 3
+
+
+@cleanup
+def test_bar_single_height():
+    # Check that a bar chart with a single height for all bars works
```

Comment:
Slightly better to do

```python
fig, ax = plt.subplots()
ax.bar(...)
```

which helps to protect from tests leaking into each other a bit more.